tests/robot-tests/tests/libs/no_javascript.py

```diff
@@ -0,0 +1,67 @@
+import requests
+from bs4 import BeautifulSoup
+
+def user_gets_parsed_html_from_page(url):
+    response = requests.get(url)
+    return BeautifulSoup(response.text, "html.parser")
+
+
+def user_checks_page_first_accordion_heading_is_not_a_button(parsed_page):
+    first_accordion = parsed_page.select_one("[data-testid='accordionSection']")
+    if not first_accordion:
+        raise AssertionError('No accordion section found')
+
+    heading = first_accordion.select_one('#publications-1-heading')
+    if not heading:
+        raise AssertionError('First accordion section heading wasn\'t found')
+    if heading.name == 'button':
+        raise AssertionError('First accordion section heading shouldn\'t be a button (with no javascript)')
+
+
+def user_checks_page_first_accordion_heading_does_not_have_attribute_aria_expanded(parsed_page):
+    first_accordion = parsed_page.select_one("[data-testid='accordionSection']")
+    if not first_accordion:
+        raise AssertionError('No accordion section found!')
+
+    heading = first_accordion.select_one('#publications-1-heading')
+    if not heading:
+        raise AssertionError('First accordion section heading wasn\'t found!')
+    if 'aria-expanded' in heading.attrs:
+        raise AssertionError('First accordion header contains "aria-expanded" attribute')
+
+
+def user_checks_page_first_accordion_section_contains_details(parsed_page):
+    first_accordion = parsed_page.select_one("[data-testid='accordionSection']")
+    if not first_accordion:
+        raise AssertionError('No accordionSection found!')
+
+    content = first_accordion.select_one('#publications-1-content')
+    if not content:
+        raise AssertionError('First accordion section content wasn\'t found!')
+
+    content_details = content.select_one('details')
+    if not content_details:
+        raise AssertionError('First accordion section content didn\'t contain details')
+
+
+def user_checks_page_first_details_summary_does_not_have_attribute_aria_expanded(parsed_page):
+    first_details = parsed_page.select_one("details")
+    if not first_details:
+        raise AssertionError('No details node found!')
+
+    summary = first_details.select_one("summary")
+    if not summary:
+        raise AssertionError('No summary node found!')
+    if 'aria-expanded' in summary.attrs:
+        raise AssertionError('First details summary contains "aria-expanded" attribute')
+
+
+def user_checks_page_first_details_contains_content(parsed_page):
+    first_details = parsed_page.select_one("details")
+    if not first_details:
+        raise AssertionError('No details node found!')
+
+    content = first_details.select_one('.govuk-details__text')
```

Comment:
This check isn't good enough. Pass in a text argument to this keyword and actually check that it's actually rendered in the details.